jib-core/src/main/java/com/google/cloud/tools/jib/http/Connection.java

```diff
@@ -66,7 +96,12 @@
    * @param url the url to send the request to
    */
   public Connection(URL url) {
```

Comment:
If we are using a `Builder` for the class, we should make all the constructors private.